modules/crm_core_default_matching_engine/config/install/crm_core_match.matcher.household.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
Looking at what's left in the default module:
* crm_core_default_matching_engine.permissions.yml sounds like it can be dropped anyway. 
* crm_core_default_matching_engine.module is all dead code too I think. 
* tests/src/Unit/Plugin/crm_core_match/engine is in a really weird place anyway :)

So other than the field match plugins/manager, nothing left. definitely seems like a good idea to just drop that module to me.